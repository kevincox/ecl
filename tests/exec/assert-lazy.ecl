local template = {
	foo = 1
	assert foo == 2
}

impl = template: {
	foo = 2
}
