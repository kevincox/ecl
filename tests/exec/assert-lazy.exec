cmd: load assert-lazy.ecl -flines
status: 0
