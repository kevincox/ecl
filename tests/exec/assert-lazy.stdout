	DICT	{
impl	DICT	{
impl.foo	NUM	2
impl	DICT	}
	DICT	}
